CONTRIBUTING.md

```diff
@@ -99,7 +99,7 @@ before you've spent a lot of time implementing it.
 
 Since Porter is a CLI, the "solution" will usually look like this:
 
-```console
+```bash
```

Comment:
There are a few reasons why we use console instead of bash

1. Very often we are demonstrating a command and it's output in the same block. We don't want the output colored as it's incorrect and distracting (like the random green sha below).

![Screen Shot 2021-05-25 at 10 36 17 AM](https://user-images.githubusercontent.com/1368985/119526663-3389e400-bd45-11eb-90ec-db26ff92b5fa.png)

2. Most code blocks are not bash specific. Porter works on all operating systems, including Windows in which case people are often using powershell.

3. Not much is added by highlighting different words of an example command differently. With console, the command is black text and the output is blue text. I have found that is a more meaningful highlighting schema for what we are doing (command+example).